.circleci/config.yml

```diff
@@ -9,12 +9,12 @@ commands:
           name: Add GitHub access token for composer
           command: echo 'composer config -g github-oauth.github.com $GITHUB_TOKEN' >> $BASH_ENV
       # Add SSH user key so we can access related repositories as part of our initial clone + composer install command.
-      # 89:cc >> GitHub user key fingerprint
-      # ed:e9 >> private fingerprint to talk to platform.sh
+      # f8:f0 >> private fingerprint to allow Circle CI to talk to platform.sh.
```

Comment:
Swapped the order as otherwise commits would end up looking like they came from me. This way, Circle CI's key takes priority.